doc/layout-config-reference.rst

```diff
@@ -117,6 +117,18 @@ MMC Controls
 MMC specific controls can be specified using the keyword ``mmc`` containing a
 mapping of the following options:
 
+``hwreset-enable`` (boolean)
```

Comment:
I choose `hwreset-enable` to avoid confusion as there is also a different hwreset disable option, which is also one-time programmable.

Reply:
Ah okay! :smiley: Looking at the mmc-utils manpage again, it now makes sense to me. Ideally, we would simple add a key "hwreset-enable" without a value, as `false` simply skips it at the moment, but the YAML spec does not allow for this.

Idea: How about we label this key "hwreset" anyway and use `false` to explicitly *disable* and `true` to explicitly *enable* the HW reset feature? If not changing this bit, the option should not be specified in the first place.

Reply:
I like the idea. But with a boolean we have no default value that does not do anything. Alternatively, the user `hwreset` could be a string which is either `enable` or `disable`.

Reply:
What's the use-case of having a default value not having any effect on the HW reset feature? E.g. if it is already enabled, then trying to enable it again should be no problem.

Reply:
There may be a misunderstanding here. What value should the internal hwreset variable have if hwreset is not set in the layout. If it is a boolean value both true and false would lead to permanent change of the eMMC in case the HW reset feature was not used before.

Reply:
I understand now. :+1: Then let's opt for the string option, as you suggested? So `enable` and `disable` for possible values to set the feature and `null` as the default.